src/components/Sidebar/Burger/index.tsx

```diff
@@ -1,23 +1,27 @@
-import { useContext } from 'react';
-import { StyledBurger } from './styles';
-import { SidebarContext } from '../../../store/sidebar-context';
+import { useContext, useEffect, useState } from "react";
+import { StyledBurger } from "./styles";
+import { SidebarContext } from "../../../store/sidebar-context";
 
 const Burger = () => {
   const { open, handleTriggerSidebar } = useContext(SidebarContext);
+  const [scrollPosition, setScrollPosition] = useState(0);
 
-  if (open) {
-    document.body.style.position = 'fixed';
-    document.body.style.top = `-${window.scrollY}px`;
-  } else {
-    document.body.style.position = '';
-    document.body.style.top = '';
-  }
+  useEffect(() => {
+    if (open) {
+      // Зберігаємо поточну позицію прокрутки при відкритті сайдбару
```

Comment:
please, remove all comments, we leave comments only with prefix TODO:
for examle
//TODO: remove after testing

Reply:
corrected